diff --git a/node_modules/react-native-keep-awake/react-native-keep-awake.podspec b/node_modules/react-native-keep-awake/react-native-keep-awake.podspec
index 95933a7..d22fd94 100644
--- a/node_modules/react-native-keep-awake/react-native-keep-awake.podspec
+++ b/node_modules/react-native-keep-awake/react-native-keep-awake.podspec
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.source         = { :git => package['repository']['url'], :tag => s.version }
 
   s.requires_arc   = true
-  s.platform       = :ios, '8.0'
+  s.platforms      = { :ios => '8.0', :tvos => '12.0' }
 
   s.preserve_paths = 'README.md', 'package.json', 'index.js'
   s.source_files   = 'ios/*.{h,m}'
